vendor/hypercore/index.d.ts

```diff
@@ -148,8 +164,30 @@ declare class Hypercore<
   readonly contiguousLength: number
   readonly fork: number
   readonly padding: number
-  static createProtocolStream(stream: boolean | Duplex | NodeDuplex | NoiseStream | ProtocolStream | ReplicationStream | Protomux, opts: CreateProtocolStreamOpts): ReplicationStream
-
+  static createProtocolStream(
+    stream: boolean,
+    opts?: CreateProtocolStreamOpts
+  ): Hypercore.ReplicationStream<Duplex>
+  static createProtocolStream(
+    stream:
+      | Duplex
+      | NodeDuplex
+      | NoiseStream
+      | ProtocolStream
+      | Hypercore.ReplicationStream
+      | Protomux<NoiseStream>,
+    opts?: CreateProtocolStreamOpts
+  ): Hypercore.ReplicationStream
+  static defaultStorage<TStorage extends typeof RandomAccessStorage>(
+    storage: TStorage
+  ): (name: string) => InstanceType<TStorage>
+  static defaultStorage<
+    TStorageFn extends (name: string) => RandomAccessStorage
+  >(storage: TStorageFn): TStorageFn
+  static defaultStorage(
+    storage: string,
+    opts: DefaultStorageOptions
```

Comment:
Not adamant about this suggestion but in case it's helpful:

```suggestion
    opts?: Partial<DefaultStorageOptions>
```